changelog/fragments/csv-allnamespaces-default.yaml

```diff
@@ -0,0 +1,5 @@
+entries:
+  - description: >
+      `generate <bundle|packagemanifests>` now generates a CSV base with only the `AllNamespaces` install mode
+      supported by default, since projects are cluster-scoped by default.
+    kind: bugfix
```

Comment:
Nit: We don't need to specify the `breaking: false` field in the fragment?

Reply:
That's the default if `breaking` isn't set.